roles/ngi_pipeline/defaults/main.yml

```diff
@@ -7,3 +7,7 @@ sthlm_config: irma_ngi_config_sthlm.yaml
 bash_env_site_script: sourceme_site.sh.j2
 bash_env_upps_script: sourceme_upps.sh
 bash_env_sthlm_script: sourceme_sthlm.sh
+
+sthlm_rna_seq_conf: {{ root_path }}/conf/ngi-rnaseq_sthlm.config
```

Comment:
This needs to be embedded in citation marks (").
Furthermore since this name is based on a name set in the rnaseq playbook, it's probably better if both of them fetch the names from host vars.